pulp_smash/constants.py

```diff
@@ -12,6 +12,13 @@
     http://pulp.readthedocs.io/en/latest/dev-guide/conventions/sync-v-async.html#call-report
 """
 
+CONTENT_SOURCE_ID = 'content-world'
```

Comment:
I checked my local Pulp 2.9 and when running the following:

```
[vagrant@dev ~]$ ls /etc/pulp/content/sources/conf.d/
```

I get no output, which means that you probably will need to set a content source. Since I don't know much about this feature, it may require to create some fixture in order to have this information.